experimental/perf/scripts/test_case_helper.py

```diff
@@ -1,35 +1,97 @@
-import os
+from pathlib import Path
 
 from bench_run_log import run_experiment
 from process_sar_helper import process_sar_files
 from setup_helper import capture_sar_metrics
 
 
-def bench_log(root_folder: str, loop_count: int, step: int):
-    for loop in range(loop_count):
-        log_folder = os.path.join(root_folder, f"loop{loop}_step{step}")
-        os.makedirs(log_folder, exist_ok=True)
-        capture_sar_metrics(log_folder)
-        run_experiment(1, step, output_file=f"{log_folder}/results.json")
-        process_sar_files(log_folder, "metrics.json")
+def bench_log(root_folder: str, loop_count: int, step=2000) -> None:
+    """Runs a set of load tests with the same parameter.
 
+    The goal is to measure the performance of a baseline test with reliable
+    and repeatable averages and max computed.
 
-def bench_log_scale_step(root_folder: str, list_of_steps: list[int]):
+    Args:
+        root_folder (str): The root directory where results will be stored.
+        loop_count (int): Number of iterations to test
+        step (int, optional): Number of step in each iteration. Defaults to 2000.
+    """
+    mc = 100
+    sort_key = 1
+
+    for loop in range(1, loop_count + 1):
+        run_experiment_helper(loop, step, mc, root_folder, sort_key)
+        sort_key += 1
```

Comment:
you could use `enumerate` for sort_key if you want

Reply:
good call